docs/advanced-config.md

```diff
@@ -16,24 +16,35 @@ To configure the Splunk Distribution of OpenTelemetry JS, you can use a combinat
-| OTEL_PROPAGATORS                     | propagators                     | `tracecontext,baggage`                | Comma-delimited list of propagators to use. Valid keys: `baggage`, `tracecontext`, `b3multi`, `b3`.
-| OTEL_SERVICE_NAME                    | serviceName                     | `unnamed-node-service`                | The service name of this Node service.
-| SPLUNK_ACCESS_TOKEN                  | accessToken                     |                                       | The optional access token for exporting signal data directly to SignalFx API.
-| SPLUNK_TRACE_RESPONSE_HEADER_ENABLED | serverTimingEnabled             | `true`                                | Enable injection of `Server-Timing` header to HTTP responses.
-| OTEL_RESOURCE_ATTRIBUTES             |                                 |                                       | Comma-separated list of resource attributes added to every reported span. <details><summary>Example</summary>`key1=val1,key2=val2`</details>
+This distribution supports all the configuration options supported by the components it uses with the defaults specified by the [OTel Specification](https://github.com/open-telemetry/opentelemetry-specification/blob/main/specification/sdk-environment-variables.md):
+
+| Environment variable<br>``startTracing()`` argument             | Default value           | Support | Notes
+| --------------------------------------------------------------- | ----------------------- | ------- | ---
+| `OTEL_ATTRIBUTE_COUNT_LIMIT`                                    |                         | Stable  | Maximum allowed span attribute count
+| `OTEL_ATTRIBUTE_VALUE_LENGTH_LIMIT`                             | `12000`\*               | Stable  | Maximum allowed attribute value size
+| `OTEL_BSP_EXPORT_TIMEOUT`                                       | `30000`                 | Stable  | Maximum allowed time to export data
+| `OTEL_BSP_MAX_EXPORT_BATCH_SIZE`                                | `512`                   | Stable  | Maximum batch size Must be less than or equal to OTEL_BSP_MAX_QUEUE_SIZE
+| `OTEL_BSP_MAX_QUEUE_SIZE`                                       | `2048`                  | Stable  | Maximum queue size
+| `OTEL_BSP_SCHEDULE_DELAY`                                       | `5000`                  | Stable  | Delay interval between two consecutive exports
+| `OTEL_EXPORTER_JAEGER_ENDPOINT`                                 | `http://localhost:14268/v1/traces` or<br>`http://localhost:9080/v1/trace`<br>if `jaeger-thrift-splunk` is used as exporter | Stable | HTTP endpoint for Jaeger traces
+| `OTEL_EXPORTER_JAEGER_PASSWORD`                                 |                         | Stable  | Password to be used for HTTP basic authentication
+| `OTEL_EXPORTER_JAEGER_USER`                                     |                         | Stable  | Username to be used for HTTP basic authentication
```

Comment:
There are currently not in the official docs (except the Jaeger endpoint env var). Are they relevant to Splunk O11y Cloud?

Reply:
:) I mean... they are configuration options for our client!

I would rather not add them here even - they're standard OTel env vars that are (eventually) supported by every distro. "eventually" is the key though - the stability and the changed defaults are the only practical reason to do it.